api/tests/e2e/spaces_test.go

```diff
@@ -177,19 +183,17 @@ var _ = Describe("Spaces", func() {
 
 		It("lists the spaces the user has role in", func() {
 			Expect(resp).To(HaveRestyStatusCode(http.StatusOK))
-			Expect(result.Resources).To(ConsistOf(
+			Expect(result.Resources).To(ContainElements(
```

Comment:
Hey @julian-hj I just made the same [change](https://github.com/cloudfoundry/cf-k8s-controllers/pull/813/files#diff-c05917f3c2675facfb5c54a733b878a228ef70c4068de32c11e69b5e1e066808R186), but didn't do your un-grouped Not Contains thing below

Reply:
cool--I think decomposing that is necessary otherwise it would only fail if all three of the unwanted spaces were in the results.

Reply:
Oh wow, I didn't know that's how it worked!

Reply:
I noticed that you also switched these tests over to use the same certUser as most of the other tests, which makes way more sense.